src/battle_anim_utility_funcs.c

```diff
@@ -1036,6 +1042,7 @@ void sub_8117F30(u8 taskId)
     DestroyAnimVisualTask(taskId);
 }
 
+// TODO: Rename. Also used by Faint Attack
```

Comment:
Let's rename it now, instead of adding a TODO

Reply:
These are for me, theyll be resolved before the PR is marked ready